docs/mining/proof-of-stake.md

```diff
@@ -26,6 +26,30 @@ You will also need to [Obtain DCR](/getting-started/obtaining-dcr.md) if you hav
 
 ---
 
+## ** How Does It Work? ** ##
+
+Purchasing a ticket for PoS is quite simple (see below) but what happens to it after you buy it?
+A ticket will go through a few stages in its lifetime:
+
+1. You buy a ticket using your wallet. The rest of the process is the same for both solo and pool tickets.
```

Comment:
Hmm yes, good point. I suppose by the time you get to buying tickets, you've already done the pool/solo setup.